### PR TITLE
SDI-119 View migration progress page

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: integration_tests/integration/startVisitsMigration.spec.ts [hunk 1]
```diff
@@ -90,8 +90,9 @@ context('Visit Migration Homepage', () => {
       page.startMigrationButton().click()
 
       const confirmationPage = Page.verifyOnPage(StartVisitsMigrationConfirmationPage)
-      confirmationPage.confirmationMessage().contains('00,988')
+      confirmationPage.confirmationMessage().contains('100,988')
```
Comment (andymarke) [inline integration_tests/integration/startVisitsMigration.spec.ts:93]: previous typo

File: server/routes/visitMigration/visitsMigrationController.ts [hunk 2]
```diff
@@ -41,26 +41,13 @@ export default class VisitsMigrationController {
...
       })
     } else {
-      const { migrations } = await this.visitMigrationService.getVisitMigrations(context(res), {
+      const { migrations } = await this.visitMigrationService.getVisitsMigrations(context(res), {
         ...searchFilter,
         toDateTime: VisitsMigrationController.withDefaultTime(searchFilter.toDateTime),
         fromDateTime: VisitsMigrationController.withDefaultTime(searchFilter.fromDateTime),
       })
 
-      const decoratedMigrations = migrations.map(migration => {
```
Comment (andymarke) [inline server/routes/visitMigration/visitsMigrationController.ts:50]: function extracted below

File: integration_tests/mockApis/nomisMigrationApi.ts [hunk 3]
```diff
@@ -214,10 +214,170 @@ const stubGetFailures = (failures: unknown = defaultFailures): SuperAgentRequest
     },
   })
 
+const stubGetMigrationDetailsStarted = (migrationId: string): SuperAgentRequest =>
+  stubFor({
```
Comment (petergphillips) [inline integration_tests/mockApis/nomisMigrationApi.ts:218]: DPS has a [getFor](https://github.com/ministryofjustice/digital-prison-services/blob/c6a8c5825780edfdb1d42e364a500eb9923d04eb/integration-tests/mockApis/wiremock.js#L46) that reduces some of this boiler plate

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**